deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
On Fri, Apr 22, 2016 at 6:03 PM, Jordan Liggitt <notifications@github.com>
wrote:
>
> Again, adding or removing labels on notes is a very highly privileged
> operation. It seems a little bit odd to make the nodes adjust to fit a
> daemonset. Does this mean we're not setting nodes up correctly?
>

@liggitt how would you suggest indicating which nodes a daemonset is
supposed to land on? The way we know of is to have a matching label on the
pod spec and on the nodes. So we label all the nodes we want the daemonset
to deploy to. This is easy for a cluster-admin, but makes an automated
upgrade (run by the deployer without such capabilities) impossible. But as
you say, node editing doesn't feel like something that should be handed to
the deployer.

So it's not that nodes are labeled wrong. It's that they require a
particular labeling for use with the fluentd daemonset, and that labeling
has to be done by a cluster-admin at some point. It adds a manual step for
the upgrade from pre-daemonset to post- but I don't really see that as a
problem.